src/utils/wizard.mts

```diff
@@ -113,23 +115,44 @@ const wizard: () => Promise<Answers> = async () => {
 		});
 	}
 
+	if (!testFileFromFlag) {
```

Comment:
By adding this, your asking for the angular test file both wether the component is Angular or not. This part should be included  in the framework specific wizard instead